menas/src/main/scala/za/co/absa/enceladus/menas/services/DatasetService.scala

```diff
@@ -26,11 +26,11 @@ import za.co.absa.enceladus.model.menas.scheduler.oozie.OozieScheduleInstance
 
 import scala.language.reflectiveCalls
 import DatasetService.RuleValidationsAndFields
-import za.co.absa.enceladus.menas.utils.enumerations.ValidationLevel
-import za.co.absa.enceladus.menas.utils.enumerations.ValidationLevel.ValidationLevel
+import za.co.absa.enceladus.utils.validation.ValidationLevel.ValidationLevel
```

Comment:
Are these used?

Reply:
Yes, the `ValidationLevel` is used. Easy to check 😉